notebooks/abbash_notebook.ipynb

```diff
@@ -1,28 +1,540 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">main_data = data[9500:11000, :]
np.savetxt('cantilever_strain.lvm', delimiter = '\t')
</pre><p>once you run this, you'll have a file called <code>cantilever_strain.lvm</code>  in your folder. You can remove the previous plotting work and load it as </p><p> </p><pre class="ql-syntax" spellcheck="false">data = np.loadtxt('./cantilever_strain.lvm')
plt.plot(data[:,0], data[:,1])
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>